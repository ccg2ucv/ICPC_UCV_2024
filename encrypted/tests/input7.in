1000
deepinsedeofthecodetheansweralwaysyoucanfindormaybenotdontworryTHISISTheLowestTHECASEfor2024Contest___________11616134wqeadafdhsavnshdfkasidfsuqscjsdndfdflkap1123853234898[[[[[[[[[]]]]]]]]]_____32312342132321324546