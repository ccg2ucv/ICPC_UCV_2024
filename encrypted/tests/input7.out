KLLWPUZLKLVM[OLJVKL[OLHUZ^LYHS^H`Z`V\JHUMPUKVYTH`ILUV[KVU[^VYY`;z0:0:;OL3V^LZ[;zwus:wMVYdbdfuVU[LZ[FFFFFFFFFFFcchchcef^XLHKHMKOZH]UZOKMRHZPKMZ\XZJQZKUKMKMSRHWccdejgedefjkjBBBBBBBBBDDDDDDDDDFFFFFedecdefdcededcedfgfh